### ISSUE TITLE
Use `bytes` base for all values

### ISSUE THREAD

**Issue by volodya-lombrozo** [objectionary/jeo-maven-plugin#715]
Currently we have different types of "value" objects in `jeo-maven-plugin` with different types:

```xml
<o base="org.eolang.jeo.int" name="access" data="bytes" line="1656645624">00 00 00 00 00 00 00 03</o>
```

```xml
<o base="org.eolang.jeo.string" data="bytes" line="844659190">28 29 56</o>
```

```xml
<o base="org.eolang.jeo.bool" data="bytes" line="1984388884">00</o>
```

New version of `eo` saves all the values as `bytes` objects. We must do the same in `jeo-maven-plugin`:

```xml
<o base="org.eolang.jeo.int" line="33" name="access" pos="4">
    <o as="0" base="bytes" data="bytes" line="34" pos="6">00 00 00 00 00 00 00 04</o>
</o>
```

**Comment by 0pdd** [comment]
@volodya-lombrozo 2 puzzles [#722](https://github.com/objectionary/jeo-maven-plugin/issues/722), [#723](https://github.com/objectionary/jeo-maven-plugin/issues/723) are still not solved.

**Comment by 0pdd** [comment]
@volodya-lombrozo the puzzle [#722](https://github.com/objectionary/jeo-maven-plugin/issues/722) is still not solved; solved: [#723](https://github.com/objectionary/jeo-maven-plugin/issues/723).

**Comment by 0pdd** [comment]
@volodya-lombrozo all 2 puzzles are solved here: [#722](https://github.com/objectionary/jeo-maven-plugin/issues/722), [#723](https://github.com/objectionary/jeo-maven-plugin/issues/723).
